sdk/script/management/account-manager.ts

```diff
@@ -816,8 +816,14 @@ export class AccountManager {
                 file = fileOrPath;
             }
 
-            req.field("package", file)
+            req.attach("package", file)
                 .field("packageInfo", JSON.stringify(packageInfo))
+                .on("progress", (event: any) => {
+                    if (uploadProgressCallback) {
+                        var currentProgress: number = (event.loaded/event.total) * 100;
```

Comment:
Remove superfluous parentheses.
Fix whitespace around '/'.
Can "event.total" ever be 0?  Perhaps when the package is an empty file.  If yes, the result would be Infinity.